SUMMARY.md

```diff
@@ -16,8 +16,8 @@ limitations under the License.
 
 # Summary
 
-* [Introduction](README.md)
-* [Quickstart Guide](QUICKSTART.md)
+* [Introduction](/README.md)
+* [Quickstart Guide](/QUICKSTART.md)
```

Comment:
These changes are not required

Reply:
I think there should be one standard for links.  At the moment we have three ways to write these links.

Reply:
we will see if this change works once the bitbooks have been compiled again

Reply:
Ok, rolled it back.